components/infobox/wikis/valorant/infobox_league_custom.lua

```diff
@@ -43,6 +50,10 @@ end
 
 ---@param args table
 function CustomLeague:customParseArguments(args)
+	self.data.platform = Cell{name = 'Platform', content = {PageLink.makeInternalLink(
```

Comment:
```suggestion
	self.data.platform = PLATFORMS[(args.platform or ''):lower()] or DEFAULT_PLATFORM
```

parse it here, do not build the display here
build the display in CutomLeague:parse